bot/discord/commands/apikey.js

```diff
@@ -1,68 +1,97 @@
-
-    let keyPool = db.get("apiKeys");
-    if (!keyPool) keyPool = [];
-
-    if (!args[0]) {
-
-        if (db.fetch(`${message.author.id}_apikey`)) {
-            return message.channel.send(`🚧 | You already have an API key. You can delete it by typing: **DBH!apikey delete**!`)
-        }
-
-        let msg = await message.channel.send(`💡 | Im getting your **API Key**...`)
-
-        try {
-
-            let embed = new Discord.MessageEmbed()
-                .setAuthor(`${client.user.username} | Api Key`, client.user.avatarURL())
-                .setDescription(`> Do not share this key with anyone else!`)
-                .addField(`🔑 | Key:`, `> \`${key}\``)
-                .addField(`❓ | Info`, `> How to Post stats? [Visit This Site](https://www.npmjs.com/package/danbot-hosting)\n> Package Github: [Click Here](https://github.com/danbot-devs/danbot-hosting)`)
-                .setColor(message.guild.me.displayHexColor)
-                .setTimestamp()
-            await message.author.send(embed)
-            msg.edit(`🔑 | Check your **DM's** for your **API Key**.`).catch((err) => {
-                message.channel.send(`🔑 | Check your **DM's** for your **API Key**.`)
-            })
-
-        } catch (err) {
-            return message.channel.send(`**An error occupied:**\n\`\`\`js\n${err}\`\`\``)
-        }
-
-        db.push("apiKeys", key);
-        db.set(`${message.author.id}_apikey`, key);
-        db.set(`${key}`, message.author.id)
-        return;
+  let keyPool = db.get("apiKeys");
+  if (!keyPool) keyPool = [];
 
+  if (!args[0]) {
+    if (db.fetch(`${message.author.id}_apikey`)) {
+      return message.channel.send(
+        "🚧 | You already have an API key. You can delete it by typing: `DBH!apikey delete` or you can view your api key by typing: `DBH!apikey viw`!"
+      );
     }
 
-    if (args[0] === "delete") {
-
-        if (!db.fetch(`${message.author.id}_apikey`)) {
-            return message.channel.send(`🚧 | You **dont** have an **API key** to delete.`)
-        }
-
-        let msg = await message.channel.send(`💡 | Im deleting your **API Key**...`);
+    let msg = await message.channel.send(`💡 | Im getting your **API Key**...`);
+
+    try {
+      let embed = new Discord.MessageEmbed()
+        .setAuthor(`${client.user.username} | Api Key`, client.user.avatarURL())
+        .setDescription(`> Do not share this key with anyone else!`)
+        .addField(`🔑 | Key:`, `> \`${key}\``)
+        .addField(
+          `❓ | Info`,
+          `> How to Post stats? [Visit This Site](https://www.npmjs.com/package/danbot-hosting)\n> Package Github: [Click Here](https://github.com/danbot-devs/danbot-hosting)`
+        )
+        .setColor(message.guild.me.displayHexColor)
+        .setTimestamp();
+      await message.author.send(embed);
+      msg.edit(`🔑 | Check your DM's for your API Key.`).catch((err) => {
+        message.channel.send(`🔑 | Check your DM's for your API Key.`);
+      });
+    } catch (err) {
```

Comment:
Instead of sending the error in the channel, notify the user that their dms are locked, since the only error which will occur here is the client failing to send a message to the user.

Reply:
i never made that changes but i changed the response.